deployment/upgradeLive

```diff
@@ -3,15 +3,32 @@
 # This uploads NVM
 set -e
 [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" || (echo "nvm not found in $NVM_DIR, this is a must, Exiting!" ; exit 1 )
+FORCE=
+REPLACE_SCRIPT=
 cd /opt/kaltura/liveController
+if [ $# -eq 2 ]; then
+ if [ "$2" = "force" ]; then
```

Comment:
I think this won't work. You're saying "if number of parameters is NOT 2 --> parameter num 2 = something"